torch/_dynamo/symbolic_convert.py

```diff
@@ -920,11 +921,22 @@ def call_function(
             raise AssertionError(f"Attempt to trace forbidden callable {inner_fn}")
         self.push(fn.call_function(self, args, kwargs))  # type: ignore[arg-type]
 
+    def inline_generator_function(self, fn, args, kwargs):
+        """
+        Redirect the call to the generator "call_function"
+        """
+        if not isinstance(fn, GeneratorFunctionVariable):
```

Comment:
Can we ever get here with fn not being a GeneratorFunctionVariable?

Reply:
Short answer is yes! This is where we wrap a (Nested)UserDefinedFunction to a GeneratorFunctionVariable